scala/BUILD

```diff
@@ -34,6 +34,8 @@ _PARSER_COMBINATORS_DEPS = ["@io_bazel_rules_scala_scala_parser_combinators"]
 
 _SCALA_XML_DEPS = ["@io_bazel_rules_scala_scala_xml"]
 
+_SCALA_SEMANTICDB_DEPS = ["@org_scalameta_semanticdb_scalac"] if SCALA_MAJOR_VERSION.startswith("2") else []
```

Comment:
please don't prefix `SemanticDB` variables with `scala` (also in other changes). Scala XML might be misleading a bit, but it has `scala` in its name.

Reply:
OK. Fixed